impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -93,6 +93,10 @@ AllocatorVK::AllocatorVK(std::weak_ptr<Context> context,
     VALIDATION_LOG << "Could not create memory allocator";
     return;
   }
+  for (auto i = 0u; i < kPoolCount; i++) {
+    created_buffer_pools_ &=
```

Comment:
Making this optional for now as the host unittests don't seem to be able to create the same buffer pools that real devices do.